packages/react-charting/src/utilities/utilities.ts

```diff
@@ -1382,5 +1382,5 @@ export const isScalePaddingDefined = (prop: number | undefined, shorthandProp?:
 
 export const formatDate = (date: Date, useUTC?: boolean) => {
   const timeFormat = useUTC ? d3UtcFormat : d3TimeFormat;
-  return timeFormat('%-e %b %Y, %H:%M')(date) + (useUTC ? ' GMT' : '');
+  return timeFormat('%-e %B %Y, %H:%M:%S')(date) + (useUTC ? ' GMT' : '');
```

Comment:
dont make the change in utilities. Make this change only in the examples below

Reply:
Okay, I will change that